lib/absinthe/logger.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Absinthe.Logger do
+  @moduledoc """
+  Handles logging of various instrumentation events.
+
+  ## Variable filtering
+
+  Absinthe can filter out sensitive information like tokens and passwords
```

Comment:
Sure (this is copied almost verbatim from Phoenix, btw, to include that wording.)